plural/plural/recipe/console-aws.yaml

```diff
@@ -40,6 +40,9 @@ sections:
   - name: console_dns
     documentation: Fully Qualified Domain Name for the console dashboard, eg console.topleveldomain.com if topleveldomain.com is the hostname you inputed above.
     type: DOMAIN
+  - name: kas_dns
```

Comment:
let's not add this yet

Reply:
Should we only add and push helm part? Will we be able to install and test it?